isp-functions/noun-deps.js

```diff
@@ -29,7 +29,7 @@ module.exports = targetValue => {
   // Split into path pieces ignoring blank/empty ones and params.
   let pieces = targetValue.split('/').filter(i => !!i);
 
-  if ((pieces.length === 1 && pieces[0] === 'search') || pieces[0] === 'me' || pieces[0] === 'cust') {
+  if ((pieces.length === 1 && pieces[0] === 'search') || pieces[0] === 'me' || pieces[0] === 'cust' || pieces.length === 6 && pieces[4] === 'preview') {
```

Comment:
Does this need parens around the two items being ANDed?

Without them I think it might evaluate every ORed expression, then AND the "aggregate" result with the final expression 🤔

Reply:
```suggestion
  if ((pieces.length === 1 && pieces[0] === 'search') || pieces[0] === 'me' || pieces[0] === 'cust' || (pieces.length === 6 && pieces[4] === 'preview')) {
```
I should have added this given how confusing/poorly I can communicate boolean logic 😢